dev_requirements.txt

```diff
@@ -1,5 +1,5 @@
 # tests
-black[jupyter] ~= 23.1
+black[jupyter] >= 24.3.0
```

Comment:
Actually, I think we should go with `~=24.3` to avoid `25.0` breaking our CI unnecessarily